### DISCUSSION TITLE
Voyage Project Submission Optional Questionnaire #1

### DISCUSSION THREAD

**Discussion by mnichols08** [chingu-voyages/v43-tier1-team-09#229] [General]
(Optional) What was the best aspect of your team project experience?

**Comment by mnichols08** [comment]
The best aspect of the team project for me was being given a prompt and the opportunity to develop a plan to build a web application with other like-minded individuals.

**Comment by PaulaR-05** [comment]
The best aspect of the program was to complete a project, gain experience and improve soft skills. We were placed in a team to work with, deadlines, other individuals, and shared goals. Also, I realized quickly that I was learning more from this Voyage than from everything I did until now. At this point, I’m grateful because it has been the best experience I’ve ever had :) Thanks a lot, everyone!
